### PR TITLE
Card Reader Module documentation

### PR DESCRIPTION
**Pull request by kidinov**
Documentation kick-off for the CPP project.

### How to test
Read the doc carefully

https://github.com/woocommerce/woocommerce-android/blob/42908d844df657c74b991540a5eb09920f16ba42/libs/cardreader/DOCUMENTATION.md

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: libs/cardreader/DOCUMENTATION.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+## Warning
+If reading this you find anything is not up-to-date, please fix it or report to the team if not sure how to do that
+
+## Useful Links To Get Started
+* Store Setup for Card Present Payment Testing - P91TBi-4BH-p2
+* Stripe terminal in-person payments - https://stripe.com/docs/terminal
+* Our very own P2 - dfdoF-p2
```
Comment (kidinov) [inline libs/cardreader/DOCUMENTATION.md]: Just noticed that these shortened links don't work for me. Could you please also click on them and see if it's the same?

P91TBi-4BH-p2 works fine for some reason

File: libs/cardreader/DOCUMENTATION.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+## Warning
```
Comment (malinajirka) [inline libs/cardreader/DOCUMENTATION.md]: We might want to consider including this in the `/docs` folder - I don't think anyone will look for a readme/documentation in the module itself. Wdyt?

File: libs/cardreader/DOCUMENTATION.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+* **Card Reader** - At the moment only one reader is supported *BBPOS Chipper 2X BT* (ask Aaron to order your own reader)
+* **COD** - Cash On Delivery. Paying with a card is also "cash", so CPP is COD 🤷
+* **POS** - Point Of Sale
+* **KYC** - Know Your Customer
+
+## CardReader Module
+
+This module provides an abstraction from a provider-specific SDK to connect and accept payments using card readers. The module currently supports only Stripe Terminal SDK for Android and most of this documentation is specific only to this SDK. However, the module tries to hide as many details as possible from the client app.
+
+### Notes
```
Comment (malinajirka) [inline libs/cardreader/DOCUMENTATION.md]: We might want to consider merging this with the Dictionary, wdyt?

File: libs/cardreader/DOCUMENTATION.md (hunk 1 above)
Comment (malinajirka) [reply libs/cardreader/DOCUMENTATION.md]: I don't use the app which turns it into a link -> it works as expected when I use wp.me manually.

**Review by malinajirka — COMMENTED** [review]
Left some comments, let me know what you think ;).

File: libs/cardreader/DOCUMENTATION.md (hunk 3 above)
Comment (kidinov) [reply libs/cardreader/DOCUMENTATION.md]: Totally - https://github.com/woocommerce/woocommerce-android/pull/4516/commits/fdaabbe82b0fe6acaab219e56bab5e8ddba12178

File: libs/cardreader/DOCUMENTATION.md (hunk 2 above)
Comment (kidinov) [reply libs/cardreader/DOCUMENTATION.md]: Yeah, moved to the docs and renamed the file https://github.com/woocommerce/woocommerce-android/pull/4516/commits/1c2e42b534905ddf93ce38c78a1d6dc84dfb736c

**Review by malinajirka — APPROVED** [review]
LGTM, thanks @kidinov! I didn't merge it since your question is still open - but feel free to merge it ;).

**Pull request merged**